postal_address/address.py

```diff
@@ -679,7 +715,9 @@ def subdivision_metadata(subdivision):
     subdiv_type_id = subdivision_type_id(subdivision)
     metadata = {
         '{}'.format(subdiv_type_id): subdivision,
-        '{}_code'.format(subdiv_type_id): subdivision.code,
+        # Rename code to slug to avoid overriding 'country_code' in some cases
+        # See https://github.com/scaleway/postal-address/issues/16
+        '{}_slug'.format(subdiv_type_id): subdivision.code,
```

Comment:
Now that I saw the use of that metadata in the wild, `_slug` might not be the right word, as it still holds a code. Don't you think `_key`, `_area_code` will be more appropriate here?